### PR TITLE
SignUp

### PR DESCRIPTION
**Pull request by Chinnu0207**

### CODE REVIEW

File: src/main/resources/db/data1.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE logins (
```
Comment (Murutechi) [inline src/main/resources/db/data1.sql:1]: please change table name to users

File: target/classes/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Manifest-Version: 1.0
+Build-Jdk-Spec: 21
```
Comment (Murutechi) [inline target/classes/META-INF/MANIFEST.MF:2]: remove this file

File: src/main/resources/db/data1.sql (hunk 1 above)
Comment (Murutechi) [inline src/main/resources/db/data1.sql:1]: please change table name to users

File: target/classes/META-INF/maven/com.jm.lms/lms-student-management/pom.properties [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#Generated by Maven Integration for Eclipse
```
Comment (Murutechi) [inline target/classes/META-INF/maven/com.jm.lms/lms-student-management/pom.properties:1]: remove it

Comment (Murutechi) [inline target/classes/META-INF/maven/com.jm.lms/lms-student-management/pom.properties:1]: remove it

File: target/classes/META-INF/MANIFEST.MF (hunk 2 above)
Comment (Murutechi) [inline target/classes/META-INF/MANIFEST.MF:2]: remove this file

File: src/main/resources/db/data1.sql (hunk 1 above)
Comment (Murutechi) [inline src/main/resources/db/data1.sql:1]: please change table name to users

File: target/classes/META-INF/maven/com.jm.lms/lms-student-management/pom.xml [hunk 4]
```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Murutechi) [inline target/classes/META-INF/maven/com.jm.lms/lms-student-management/pom.xml:1]: Remove it

File: target/classes/application.properties [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Database
```
Comment (Murutechi) [inline target/classes/application.properties:1]: remove it

File: target/classes/db/data.sql [hunk 6]
```diff
@@ -0,0 +1,14 @@
+INSERT INTO STUDENTS (id, first_name, last_name, date_of_birth, email) VALUES 
+					 (1, 'Ashley', 'Miller', '1999-12-01', 'ashleymiller@gmail.com');
```
Comment (Murutechi) [inline target/classes/db/data.sql:2]: remove it

File: target/classes/db/data1.sql [hunk 7]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE logins (
```
Comment (Murutechi) [inline target/classes/db/data1.sql:1]: remove it

File: target/classes/db/schema.sql [hunk 8]
```diff
@@ -0,0 +1,8 @@
+create table students (
```
Comment (Murutechi) [inline target/classes/db/schema.sql:1]: Remove it

File: target/classes/langs/messages.properties [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Student Validation (+ Email)
```
Comment (Murutechi) [inline target/classes/langs/messages.properties:1]: Remove it

File: target/maven-archiver/pom.properties [hunk 10]
```diff
@@ -0,0 +1,3 @@
+artifactId=lms-student-management
```
Comment (Murutechi) [inline target/maven-archiver/pom.properties:1]: Remove it

File: target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst [hunk 11]
```diff
@@ -0,0 +1,11 @@
+com\jm\lms\studentms\exception\StudentNotFoundException.class
```
Comment (Murutechi) [inline target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst:1]: Remove it

File: target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst [hunk 12]
```diff
@@ -0,0 +1,11 @@
+C:\Users\nandi\feature-signup\lms-app\src\main\java\com\jm\lms\studentms\service\StudentService.java
```
Comment (Murutechi) [inline target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst:1]: Remove it.

**Review by Murutechi — COMMENTED** [review]
It seems there are multiple conflicts in this branch . Please kindly resolve it asap.

File: .project [hunk 13]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
```
[the remark this answers is not in the document]
Comment (Chinnu0207) [reply .project]: done sir

File: .settings/org.eclipse.core.resources.prefs [hunk 14]
```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
```
[the remark this answers is not in the document]
Comment (Chinnu0207) [reply .settings/org.eclipse.core.resources.prefs]: done sir

File: .settings/org.eclipse.jdt.apt.core.prefs [hunk 15]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
[the remark this answers is not in the document]
Comment (Chinnu0207) [reply .settings/org.eclipse.jdt.apt.core.prefs]: done sir

File: .settings/org.eclipse.jdt.core.prefs [hunk 16]
```diff
@@ -0,0 +1,10 @@
+eclipse.preferences.version=1
```
[the remark this answers is not in the document]
Comment (Chinnu0207) [reply .settings/org.eclipse.jdt.core.prefs]: done sir

File: .settings/org.eclipse.m2e.core.prefs [hunk 17]
```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```
[the remark this answers is not in the document]
Comment (Chinnu0207) [reply .settings/org.eclipse.m2e.core.prefs]: done sir

File: src/main/java/com/jm/lms/studentms/controller/SignUpServiceController.java [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+import com.jm.lms.studentms.service.SignUpService;
+
+@RestController
+@RequestMapping("api/v1/lms/signup")
+public class SignUpServiceController {
+
+@Autowired
+private SignUpService signUpService;
+
+@PostMapping("/add")
```
[the remark this answers is not in the document]
Comment (Chinnu0207) [reply src/main/java/com/jm/lms/studentms/controller/SignUpServiceController.java]: Done sir

File: src/main/java/com/jm/lms/studentms/model/SignUp.java [hunk 19]
```diff
@@ -0,0 +1,85 @@
...
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "logins")
```
[the remark this answers is not in the document]
Comment (Chinnu0207) [reply src/main/java/com/jm/lms/studentms/model/SignUp.java]: Done sir

File: src/main/java/com/jm/lms/studentms/service/SignUpService.java [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SignUpService {
+
+    @Autowired
+    private SignUpServiceRepository signUpServiceRepository;
+
+    public String saveRegister(SignUp signUp) {
```
[the remark this answers is not in the document]
Comment (Chinnu0207) [reply src/main/java/com/jm/lms/studentms/service/SignUpService.java]: Done sir

File: src/main/resources/db/data1.sql (hunk 1 above)
Comment (Chinnu0207) [reply src/main/resources/db/data1.sql]: Done sir

File: target/classes/META-INF/MANIFEST.MF (hunk 2 above)
Comment (Chinnu0207) [reply target/classes/META-INF/MANIFEST.MF]: Done sir

File: target/classes/META-INF/maven/com.jm.lms/lms-student-management/pom.properties (hunk 3 above)
Comment (Chinnu0207) [reply target/classes/META-INF/maven/com.jm.lms/lms-student-management/pom.properties]: Done sir

File: target/classes/META-INF/MANIFEST.MF (hunk 2 above)
Comment (Chinnu0207) [reply target/classes/META-INF/MANIFEST.MF]: Done sir

File: src/main/resources/db/data1.sql (hunk 1 above)
Comment (Chinnu0207) [reply src/main/resources/db/data1.sql]: Done sir

File: target/classes/META-INF/maven/com.jm.lms/lms-student-management/pom.xml (hunk 4 above)
Comment (Chinnu0207) [reply target/classes/META-INF/maven/com.jm.lms/lms-student-management/pom.xml]: Done sir

File: target/classes/application.properties (hunk 5 above)
Comment (Chinnu0207) [reply target/classes/application.properties]: Done sir

File: target/classes/db/data.sql (hunk 6 above)
Comment (Chinnu0207) [reply target/classes/db/data.sql]: Done sir

File: target/classes/db/data1.sql (hunk 7 above)
Comment (Chinnu0207) [reply target/classes/db/data1.sql]: Done sir

File: target/classes/db/schema.sql (hunk 8 above)
Comment (Chinnu0207) [reply target/classes/db/schema.sql]: Done sir

File: target/classes/langs/messages.properties (hunk 9 above)
Comment (Chinnu0207) [reply target/classes/langs/messages.properties]: Done sir

File: target/maven-archiver/pom.properties (hunk 10 above)
Comment (Chinnu0207) [reply target/maven-archiver/pom.properties]: Done sir

File: target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst (hunk 11 above)
Comment (Chinnu0207) [reply target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst]: Done sir

File: target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst (hunk 12 above)
Comment (Chinnu0207) [reply target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst]: Done sir

**Pull request closed without merging**